contracts/governance/Vesting/VestingLogic.sol

```diff
@@ -140,7 +141,11 @@ contract VestingLogic is IVesting, VestingStorage, ApprovalReceiver {
      * @param isGovernance Whether all tokens (true)
      * or just unlocked tokens (false).
      * */
-    function _withdrawTokens(address receiver, bool isGovernance) internal {
+    function _withdrawTokens(
```

Comment:
this implementation doesn't solve the block gas limit issue - it should be refactored to allow partial withdrawals using [startFrom, maxIterations] range pattern 
because vesting contracts are not upgradable, i suggest adding these as parameters to the function call rather than adding to the storage because changing them will be tricky

Reply:
updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/529/commits/c3979c0614ca2a85fd4e60aae785d2ff16cef7de)